experimental/rsocket-test/RSocketClientServerTest.cpp

```diff
@@ -1,32 +1,61 @@
 // Copyright 2004-present Facebook. All Rights Reserved.
 
+#include <random>
 #include <utility>
 
-#include <folly/io/IOBuf.h>
 #include <gmock/gmock.h>
 
 #include "handlers/HelloStreamRequestHandler.h"
 #include "rsocket/RSocket.h"
 #include "rsocket/transports/TcpConnectionAcceptor.h"
+#include "rsocket/transports/TcpConnectionFactory.h"
 
 using namespace rsocket;
 using namespace rsocket::tests;
 
+namespace {
+
+std::random_device device;
+std::uniform_int_distribution<uint16_t> dis(9000, 10000);
+
+// Helps prevent against port collisions.
+uint16_t randPort() {
```

Comment:
Instead of this you can use port 0 to have the kernel assign a port (and read it back).